test/apps/iodemo/io_demo.cc

```diff
@@ -1975,6 +1977,16 @@ class DemoClient : public P2pDemoCommon {
             LOG << "invalid address " << server_addr;
             abort();
         }
+        if (opts().local_addr != NULL) {
+            memset(&local_addr, 0, sizeof(local_addr));
```

Comment:
Why do we need the memset? Remove it if not really needed

Reply:
This was remained same as connect_addr. As sin_port in it should be set to 0.